tests/drivers/clock_control/stm32_clock_configuration/stm32u5_core/testcase.yaml

```diff
@@ -24,3 +24,7 @@ tests:
     extra_args: DTC_OVERLAY_FILE="boards/clear_clocks.overlay;boards/pll_hse_160.overlay"
     # Build only as HSE not implemened on available boards
     build_only: true
+  drivers.stm32_clock_configuration.u5.sysclksrc_pll_hse_fracn_160:
```

Comment:
What about using HSI as PLL input which would allow a "runable" test ?

Reply:
I can add a test case using HSI with a fraction value of 0. Just to make sure that it still generates the correct PLL output. I will think about any other cases later. But for now, I can only suggest using HSI with a fraction value of 0. Perhaps, you have some test cases in mind?

Reply:
Let's go with 0